mteb/tasks/Retrieval/slk/SKQuadRetrieval.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from datasets import load_dataset
+
+from mteb.abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SKQuadRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
```

Comment:
Please add citation.

Reply:
Is there no citation for this dataset? I'm not sure if that aligns with our goals if there isn't any. cc @KennethEnevoldsen

Reply:
Then simply add:

citation=""

Reply:
Will use empty string.